pkg/doc/verifiable/common_test.go

```diff
@@ -238,6 +238,7 @@ func TestDIDKeyResolver_Resolve(t *testing.T) {
 	didDoc := createDIDDoc()
 	publicKey := didDoc.PublicKey[0]
 	authentication := didDoc.Authentication[0]
+	assertionMethod := didDoc.AssertionMethod[0]
```

Comment:
what about the other sections?

Reply:
I tested all the verification methods in `did` unit tests.